recipe/meta.yaml

```diff
@@ -18,14 +18,15 @@ requirements:
     - python >=3.5
     - pip
   run:
-    - python >=3.5
-    - cartopy
+    - python
     - geopy
+    - matplotlib
     - mercantile
     - pandas
     - pillow
     - rasterio
     - requests
+    - six
```

Comment:
six is not needed? (since it's py3 only)

Reply:
But you need to add joblib. See https://github.com/darribas/contextily/blob/master/requirements.txt for the exact list